modules/installation-launching-installer.adoc

```diff
@@ -303,6 +303,11 @@ ifdef::vsphere,vmc[]
 The installation program connects to your vCenter instance.
 .. Select the datacenter in your vCenter instance to connect to.
 .. Select the default vCenter datastore to use.
++
+[NOTE]
+====
+Datastore and cluster names cannot exceed 60 characters; therefore, ensure the combined string length does not exceed the 60 character limit when picking datastore and cluster names.
```

Comment:
Is the text "when picking datastore and cluster names" necessary?

Reply:
Thanks, @nalhadef and I agree! Adding "when picking datastore and cluster names" is not necessary as it is mentioned in the beginning part of the sentence. 

I have incorporated the above suggested changes! :)